uai2017/python/precreate_profiles.py

```diff
@@ -0,0 +1,144 @@
+import openreview
```

Comment:
can we move the script to the scripts folder ? it is not UAI conference related.